os/arch/arm/src/armv7-m/up_mpu.c

```diff
@@ -341,8 +341,15 @@ void mpu_user_extsram_context(uint32_t region, uintptr_t base, size_t size, uint
 #ifdef CONFIG_APPS_RAM_REGION_SHAREABLE
 			MPU_RASR_S |                   /* Shareable     */
 #endif
-			MPU_RASR_C |                   /* Cacheable     */
-			MPU_RASR_AP_RWRW;              /* P:RW   U:RW   */
+			MPU_RASR_C;                   /* Cacheable     */
+	if (readonly)
```

Comment:
Please use braces even it has a single statement. It's our coding rule.

Reply:
Ok. done